src/gatsby/src/components/park/reservationsRequired.js

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { Link } from "gatsby"
+
+export default function ReservationsRequired({
+  subAreas, hasDayUsePass, hasReservations, hasBackcountryPermits, hasBackcountryReservations
+}) {
+  const frontcountryCodes = ["frontcountry-camping", "group-camping", "walk-in-camping", "cabins-huts"]
+  const backcountryCodes = ["backcountry-camping", "wilderness-camping"]
+  const backcountryPermitCode = ["backcountry-camping"]
+  const groupCampingCode = ["group-camping"]
+  const picnicCode = ["picnic-shelters"]
+  const hasFrontcountry = subAreas.some(s => frontcountryCodes.includes(s.facilityType.facilityCode))
```

Comment:
I was getting an error when I did a gatsby build on my local.  I fixed it by changing the query to `.includes(s.facilityType?.facilityCode))` on this line and the next 4 lines as well.